src/gromacs/nblib/gmxcalculator.h

```diff
@@ -80,6 +81,8 @@ class GmxForceCalculator
     //! Non-Bonded Verlet object for force calculation
     std::unique_ptr<nonbonded_verlet_t> nbv_;
```

Comment:
@kanduri, I think that this object is a manager that encapsulates the NBV object. Therefore, I believe this shouldn't be public, right?

Reply:
Done.